src/components/VxpAccordion.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <StackLayout class="accordion">
+    <slot></slot>
+  </StackLayout>
+</template>
+
+<script>
+import StackLayout from '../layouts/StackLayout';
+
+export default {
+  name: 'VxpAccordion',
+  components: { StackLayout },
+  props: {
+    initialExpandedItems: {
+      required: false,
+      type: Array,
+      default: function() {
+        return [];
+      },
+      validator: items => Array.isArray(items) && !items.some(n => typeof n !== 'number'),
+    },
+    autoCollapse: {
+      required: false,
+      type: Boolean,
+      default: false,
+    },
+  },
+  mounted() {
+    this.$on('child-registered', child => {
```

Comment:
this seemed dangerous and confusing to me. I think children should trigger events like this.emit("child-registered") and this component should listen these events like <slot @child-registered>.

In current situation we listen an event that we dont know the source.